autocomplete/bash_autocomplete

```diff
@@ -3,14 +3,19 @@
 : ${PROG:=$(basename ${BASH_SOURCE})}
 
 _cli_bash_autocomplete() {
+  if [[ "${COMP_WORDS[@]:0:$COMP_CWORD}" != "source" ]]; then
     local cur opts base
     COMPREPLY=()
     cur="${COMP_WORDS[COMP_CWORD]}"
-    opts=$( ${COMP_WORDS[@]:0:$COMP_CWORD} --generate-bash-completion )
+    if [[ $cur == -* ]]; then
+      opts=$( ${COMP_WORDS[@]:0:$COMP_CWORD} ${cur} --generate-bash-completion )
```

Comment:
Some shellcheck hint about this:
`SC2068: Double quote array expansions to avoid re-splitting elements.`
See https://github.com/koalaman/shellcheck/wiki/SC2068

Reply:
done